CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# See https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file
+#
+*                   @equinix/governor-devrel-engineering
+/cmd/migration-tool @equinix/governor-devrel-engineering
```

Comment:
is this one required? previous wildcard covers that pattern

Reply:
I expect the wildcard could change over time and wanted to call out the specific add-on

Reply:
Maybe out of scope for this PR, but at what point does the migration tool go away?

Reply:
Removal would satisfy #295